### PR TITLE
Sdks 3186 move android samples

### PR DESCRIPTION
**Pull request by ryanbas21**

### CODE REVIEW

**Review by witrisna — COMMENTED** [review]
Do not inclide .gradle and .idea in the repo

**Review by witrisna — COMMENTED** [review]
Use dummy google-services.json

**Comment by george-bafaloukas-forgerock** [comment]
> google-services.json

I copied the projects as is. Do we have a dummy version to use?

**Comment by forgerock-chris** [comment]
Does GitHub squash the commits before merging? If not, can we please?
Make it a bit cleaner if it just has a single "Added Android examples" commit.

File: README.md [hunk 1]
```diff
@@ -7,39 +7,33 @@
...
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
 
-- [**Embedded login - `embedded-login`**](./embedded-login/README.md)
+To try out the Ping JavaScript SDK please look at one of our samples under [javascript](./javascript/)
 
-  - A barebones example of how to use the `@forgerock/javascript-sdk` in a vanilla html & javascript app with embedded login.
+To try out the Ping iOS SDK please look at one of our samples under [iOS](./iOS/)
 
-- [**Central login - `central-login`**](./central-login/README.md)
+To try out the Ping iOS SDK please look at one of our samples under [android](./android/)
```
Comment (forgerock-chris) [inline README.md:18]: iOS -> Android

File: android/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Samples
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping Android SDK please look at one of our samples:
+
+- [**Auth Sample - `/java-authsample`**](./java-authsample/)
```
[the remark this answers is not in the document]
Comment (forgerock-chris) [reply android/README.md:16]: Or, if we're no longer advocating/supporting the auth-ui module, should we kill these?

File: iOS/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping iOS SDK please look at one of our samples:
+
+- [**FR Example - `/uikit-frexamples/FRExample`**](./uikit-frexamples/)
+  
+  - An example iOS project making full use of the Ping SDK with a prototyping UI. Allows to explore the SDK uses in detail with example calls for running Journeys, getting OAuth2.0 tokens, User Info and more. Running the workspace makes use of the `FRExample`, `FRExampleObjC` and `FRAuthenticatorExample` projects.
+  (https://backstage.forgerock.com/docs/sdks/latest/sdks/tutorials/ios/index.html)
+
+- [**Passkeys Sample - `/uikit-passkeys-sample`**](./uikit-passkeys-sample/UnsummitAuthentication)
```
Comment (forgerock-chris) [inline iOS/README.md:21]: In a "samples" repo, we can probably do away with appending `-sample` on the folder name (they are all samples after all)
`uikit-passkeys` is plenty.

File: android/java-authenticator/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/sdk-sample-apps">
+    <img src="https://cdn.forgerock.com/logo/interim/Logo-PingIdentity-ForgeRock-Hor-FullColor.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+## Samples
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:8]: Authenticator app sample

File: android/java-authenticator/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:12]: Could be something like:
### Integrate with MFA capabilities on PingOne Advanced Identity Cloud / PingAM:

File: android/java-authenticator/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+To try out the Ping Android SDK please look at one of our samples:
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:14]: Replace with something like:
This Authenticator sample app supports registration of multiple accounts and multiple different authentication methods in each account, such as push notifications and one-time passwords.

File: android/java-authenticator/README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+WIP
+
+
+### Integrate with PingOne DaVinci:
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:19]: Remove this section

File: android/java-authenticator/README.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+- Samples coming soon
+
+## Requirements
+
+iOS
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:25]: Remove

File: android/java-authenticator/README.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+- Latest version of Android Studio
```
Comment (rodrigoareis) [inline android/java-authenticator/README.md:26]: Add to requirements:
Ping Advanced Identity Cloud or Ping AM 7.1 and above
Android API level 23+
Google Firebase project (google-services.json)

File: android/java-authenticator/README.md
Comment (rodrigoareis) [inline android/java-authenticator/README.md:1]: Would be nice to adde a Getting Started section as well with a content as follow.

To try out the ForgeRock Android SDK sample, perform these steps:

Setup an Journey instance with OATH and/or Push nodes.
Clone this repo
Open this Android project in [Android Studio](https://developer.android.com/studio).
Add your `google-services.json` to the app folder.
On the Run menu, click Run 'app'.

File: javascript/embedded-login/src/main.js
Comment (forgerock-chris) [inline javascript/embedded-login/src/main.js:1]: On line 16, please append `.html` to the example value in the comment.

E.g.
redirectUri: `${window.location.origin}/callback.html`, // e.g. 'https://sdkapp.example.com:8443/callback.html'

Many thanks!!

**Comment by george-bafaloukas-forgerock** [comment]
New PR has been raised

**Pull request closed without merging**